### ISSUE TITLE
[BUG] Rebase Main Branch w/ Refactors from upstream core

### ISSUE THREAD

**Issue by nknize** [opensearch-project/performance-analyzer#516]
**What is the bug?**
Build fails with some latest namespace and library refactoring from core (e.g., https://github.com/opensearch-project/OpenSearch/pull/9005). This was caught by the [compatibility checker](https://github.com/opensearch-project/OpenSearch/actions/runs/5742101583/job/15563536027?pr=9073#step:3:2060) on https://github.com/opensearch-project/OpenSearch/pull/9073.

**How can one reproduce the bug?**
Build with the latest core snapshot

**What is the expected behavior?**
Build should pass without any "Class Not Found" or "package does not exist" failures.

**Do you have any screenshots?**
```
Checking compatibility for: https://github.com/opensearch-project/performance-analyzer.git with ref: main

2023-08-02T18:11:21.0603693Z FAILURE: Build failed with an exception.
2023-08-02T18:11:21.0610134Z 
2023-08-02T18:11:21.0616763Z * What went wrong:
2023-08-02T18:11:21.0657754Z Execution failed for task ':updateShas'.
2023-08-02T18:11:21.0665800Z > Could not resolve all dependencies for configuration ':runtimeClasspath'.
2023-08-02T18:11:21.0671961Z    > Conflict(s) found for the following module(s):
2023-08-02T18:11:21.0685566Z        - org.apache.logging.log4j:log4j-api between versions 2.20.0 and 2.17.1
2023-08-02T18:11:21.0690490Z        - org.apache.logging.log4j:log4j-core between versions 2.20.0 and 2.17.1
2023-08-02T18:11:21.0695003Z      Run with:
2023-08-02T18:11:21.0702739Z          --scan or
2023-08-02T18:11:21.0708545Z          :dependencyInsight --configuration runtimeClasspath --dependency org.apache.logging.log4j:log4j-api
2023-08-02T18:11:21.0728976Z      to get more insight on how to solve the conflict.
2023-08-02T18:11:21.0729521Z 
2023-08-02T18:11:21.0737442Z * Try:
2023-08-02T18:11:21.0748836Z > Run with --stacktrace option to get the stack trace.
2023-08-02T18:11:21.0749759Z > Run with --info or --debug option to get more log output.
2023-08-02T18:11:21.0764745Z > Run with --scan to get full insights.
```

**Do you have any additional context?**
Add any other context about the problem.

[#5910](https://github.com/opensearch-project/OpenSearch/issues/5910)

**Comment by gaiksaya** [comment]
Related #500

**Comment by khushbr** [comment]
Fixed at https://github.com/opensearch-project/performance-analyzer-commons/commit/0197ef09386c5517073a33268db0fc1612e9c1a8
